azurelinuxagent/distro/suse/rdma.py

```diff
@@ -0,0 +1,82 @@
+# Microsoft Azure Linux Agent
+#
+# Copyright 2014 Microsoft Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Requires Python 2.4+ and Openssl 1.0+
+#
+
+import glob
+import os
+import azurelinuxagent.logger as logger
+import azurelinuxagent.utils.shellutil as shellutil
+from azurelinuxagent.distro.default.rdma import RDMAHandler
```

Comment:
I don't think this will resolve - do we need to move the RDMA base and this one under common somewhere?